eng/scripts/Language-Settings.ps1

```diff
@@ -48,4 +48,7 @@ function Publish-c-GithubIODocs ($DocLocation, $PublicArtifactLocation)
     $pkgInfo = Get-Content $DocLocation/package-info.json | ConvertFrom-Json
     $releaseTag = RetrieveReleaseTag "C" $PublicArtifactLocation
     Upload-Blobs -DocDir $DocLocation -PkgName 'docs' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
```

Comment:
Lets go ahead and remove the docs version.

Reply:
Done